src/modules/account/components/Overview/overview.js

```diff
@@ -44,6 +39,7 @@ const Overview = ({ tokenData, setFilter }) => {
           select={(selectedValue, option) => selectedValue?.tokenName === option.tokenName}
           className={styles.menuWrapper}
           popupClassName={styles.popupWrapper}
+          disabled={isFetching}
         >
           {tokens.map((tokenInfo) => (
```

Comment:
NRTTL
We should probably hide this wrapper if no items? Like when on Betanet 🤔 
<img width="500" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/32b67378-79be-4235-9bde-1ac242070e32">